src/Containers.jsx

```diff
@@ -243,7 +222,7 @@ const ContainerActions = ({ container, healthcheck, onAddNotification, localImag
         }
         if (container.isSystem && container.State?.CheckpointPath) {
             actions.push(
-                <DropdownSeparator key="separator-0" />,
+                <Divider key="separator-0" />,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1605-20240306-082630-f936f937-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)